.github/workflows/ci.yaml

```diff
@@ -16,89 +16,85 @@ jobs:
     name: Build
     runs-on: ubuntu-latest-8-cores
     steps:
-      - name: Checkout
+      - name: checkout
         uses: actions/checkout@v4
       - uses: extractions/setup-just@v1
       - uses: actions/cache@v3
-        name: Cache
+        name: cache
         with:
           path: |
             ~/.cargo/
             target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
-      - name: Build
+      - name: build
         run: just build
```

Comment:
musing: I have a preference to not name these steps, since they're 1-liner commands we know and often run. The idea here is that `build` is no better than `just build`, while adding an alias/mapping layer.

I had reflexively gotten into a habit of always naming each step, as I thought you had to, but it turns out that if you only provide `- run: just build` the name will also be `just build`.

This is opinionated and preferential, and I'm making this comment just for sharing, and in the case that it feels helpful in this repo.

Reply:
yeah, makes sense. I did a bunch of clean up here just to lowercase things, and I feel less strongly.